apps/src/applab/Exporter.js

```diff
@@ -569,7 +569,7 @@ export default {
       sessionId: `${getEnvironmentPrefix()}-${project.getCurrentId()}`,
       files,
       name: project.getCurrentName(),
-      sdkVersion: '25.0.0',
+      sdkVersion: '28.0.0',
```

Comment:
Do you have a sense of how soon we'll have to address the issue preventing us from upgrading past v28?

v32 - January 2019
v31 - November 2018
v29 - July 2018

Having trouble finding a complete history, but it looks like they're releasing a major version every two months?  So we should assume we have six months to upgrade again?

Reply:
Yes, that sounds about right. I hope to discuss with them soon to confirm.